changelog/std-file-readText.dd

```diff
@@ -1,7 +1,7 @@
 readText now checks BOMs
 
 $(REF readText, std, file) now checks for a
-$(HTTP https://en.wikipedia.org/wiki/Byte_order_mark, BOM). If a BOM is present
+$(HTTPS en.wikipedia.org/wiki/Byte_order_mark, BOM). If a BOM is present
```

Comment:
Ideally we phase out this entire HTTP/HTTPS non-sense, and just use LINK2, but Andrei has been against this in the past.

Reply:
breaking down links like that is UGLY and contrary to having readable docs when they're un-rendered (eg we can't simply copy paste these links to/from a browser); and it's error prone as examplified here

Reply:
See also:

https://github.com/dlang/phobos/pull/4411#issuecomment-226326952
https://github.com/dlang/phobos/pull/4951#issuecomment-266670581

In short:
- at least there's `WEB` anymore
- The Ddoc issue has been fixed, so in theory we could try this again, but I'm not sure whether it's worth the pain.

Though maybe we can can @andralex's approval this time?

Reply:
Typing `https://` is awkward and integrating it in a macro is sensible. We could and should fix matters on the ddoc side; the issue of line breakage in macros is not singular to this one. Until then please leave HTTP and HTTPS where they are. Thanks.